src/MediaWiki/Hooks/SkinAfterContent.php

```diff
@@ -50,6 +50,10 @@ public function process() {
 
 	private function canPerformUpdate() {
 
+		if ( !$this->skin instanceof Skin ) {
```

Comment:
`! ( $this->skin instanceof Skin )`

I think that form is nicer, since otherwise the ! and the first part look like they are grouped.